index.html

```diff
@@ -187,7 +187,12 @@ <h2>Verification</h2>
       <em>This section is non-normative.</em>
 
       <p>
-TBD
+<a>Verification</a> is the process a <a>verifier</a> or <a>holder</a> undergoes
```

Comment:
`undergoes`
-> `performs`
(The VC undergoes this process.)

Reply:
fixed in [058b883](https://github.com/w3c/vc-imp-guide/pull/40/commits/058b883d0adafa4a54c8c49fddce42146f84ba31)